mmv1/third_party/terraform/services/compute/resource_compute_region_instance_group_manager.go.tmpl

```diff
@@ -1180,6 +1283,30 @@ func expandDistributionPolicyForCreate(d *schema.ResourceData) *compute.Distribu
   return distributionPolicy
 }
 
+{{ if ne $.TargetVersionName `ga` -}}
+func flattenInstanceFlexibilityPolicy(instanceFlexibilityPolicy *compute.InstanceGroupManagerInstanceFlexibilityPolicy) []map[string]any {
+	results := []map[string]any{}
```

Comment:
results -> flattenedInstanceFlexibilityPolicies?

Reply:
Tried to modify all of the naming appropriately instead of having short name.